File: src/me/Qball/Wild/Utils/CheckConfig.java
package me.Qball.Wild.Utils;

import java.util.ArrayList;
import java.util.List;

import me.Qball.Wild.Wild;
import org.bukkit.potion.PotionEffectType;

public class CheckConfig {
    Wild wild = Wild.getInstance();


    public boolean isCorrectPots() {
        ArrayList<String> potions = new ArrayList<>();
        List<String> pots = wild.getListPots();
        for(PotionEffectType effect : PotionEffectType.values())
            potions.add(effect.getName());
        for (String s : pots) {
            String[] pot = s.split(":");
            if (pot.length != 2 || !potions.contains(pot[0])) {
                return false;
            }
        }

        return true;
    }
} 

File: src/main/kotlin/me/qball/wild/util/CheckConfig.kt
package me.qball.wild.util

import me.qball.wild.Wild

object CheckConfig {

    private val plugin = Wild.instance

    fun isCorrectPots() : Boolean {
        var wrong = 0
        val pots = plugin.getListPots()
        for(i in 0..pots.size) {
            val pot = pots.get(i).split(":")
            if(pot.size != 2) {
                wrong++
                break
            }
        }
        return wrong == 0
    }
}


